build-system/tasks/visual-diff/index.js

```diff
@@ -485,26 +436,25 @@ async function generateSnapshots(percy, webpages) {
     await page.goto(
       `http://${HOST}:${PORT}/examples/visual-tests/blank-page/blank.html`
     );
-    await percy.snapshot('Blank page', page, SNAPSHOT_SINGLE_BUILD_OPTIONS);
+    await percySnapshot(page, 'Blank page', SNAPSHOT_SINGLE_BUILD_OPTIONS);
   }
 
-  log('verbose', 'Generating snapshots...');
-  if (!(await snapshotWebpages(percy, browser, webpages))) {
+  log('info', 'Generating snapshots...');
+  if (!(await snapshotWebpages(browser, webpages))) {
     log('fatal', 'Some tests have failed locally.');
   }
```

Comment:
The old library used to print the build URL at the end. I found it really useful while looking at Travis logs. Is there a way to do that here? Something like:

```js
  if (await snapshotWebpages(browser, webpages)) {
    log('info', 'Percy is now processing. Results can be found at: ' + buildUrl);
  } else {
    log('fatal', 'Some tests have failed locally.');
  }
```

Reply:
The agent should print the build URL when it exits. The URL is missing because the agent didn't exit successfully